### ISSUE TITLE
Example missing for search box

### ISSUE THREAD

**Issue by GuillaumeLeclerc** [angular-ui/angular-google-maps#1554]
Hello

It seems the pluker in this page is broken : http://angular-ui.github.io/angular-google-maps/#

Thank you for your project !

Guillaume Leclerc

**Comment by samirfor** [comment]
+1

**Comment by kandizzy** [comment]
Here's a link a direct link to Plunkrfor now until we fix the ones in the docs
http://plnkr.co/edit/Qsm9KgByM24i1QhLbTfK?p=preview

There are also multiple searchbox examples in the examples folder if you need.

**Comment by nmccready** [comment]
@kandizzy fyi the examples are mostly functional again. I fixed the dependencies on the website branch. So the example itself is the only thing to fix now.

**Comment by nmccready** [comment]
see http://angular-ui.github.io/angular-google-maps/#!/api/search-box

**Comment by kandizzy** [comment]
Cool, it doesn't seem to load the places library- been meaning to look at for a while now. Will try to resolve it asap

**Comment by nmccready** [comment]
Yeah they might have removed it; recently as the WeatherLayer is gone as well.

**Comment by nmccready** [comment]
Would have been nice w/ a bump to 4.0 google maps.

**Comment by samirfor** [comment]
I want to do just that: https://google-developers.appspot.com/maps/documentation/javascript/examples/full/places-searchbox

**Comment by SaidThaher** [comment]
hi
When you will fix the search box example in pluker?? 
the search functionality not working
